README.md

```diff
@@ -516,15 +516,15 @@ Create a database.
 
 USAGE
   $ fauna create-database DBNAME [--url <value>] [--timeout <value>]
-    [--secret <value>] [--endpoint <value>] [--stack <value>]
+    [--secret <value>] [--endpoint <value>] [--environment <value>]
 
 ARGUMENTS
   DBNAME  database name
 
 FLAGS
   --endpoint=<value>  Connection endpoint, from ~/.fauna-shell
   --secret=<value>    Secret key. Overrides the `secret` in ~/.fauna-shell
-  --stack=<value>     Stack to use, from a Fauna project
+  --environment=<value>     environment to use, from a Fauna project
```

Comment:
the README should be generated with the `readme.py` script, which will update this from the `--help` output.